### DISCUSSION TITLE
Outline around `Text2dBundle`?

### DISCUSSION THREAD

**Discussion by jamesperez2005** [bevyengine/bevy#12185] [Q&A]
I would like to spawn a `Text2dBundle` with a specific font color and size, but have a stroke outline around the text to emphasize it. Is that possible in Bevy? I tried spawning a `(Text2dBundle, Outline::new(...))` but that seems to not have the desired effect.

**Comment by nicopap** [comment]
It is currently impossible in bevy, but someone has been developping a plugin, you should hook up with that person:

* Someone working on text outline: [discord link](https://discord.com/channels/691052431525675048/866787577687310356/1209242430587543562)
* See this issue for more informations: https://github.com/bevyengine/bevy/issues/7210
